modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -3,4 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Organization Administrator of the master organization, after you approve a client app to access your API, you can access information about the app from API Manager > Contracts. You can view and reset the unique client ID and client secret for the application. When calling the API, the client app owner needs to pass these credentials if the API is protected with policies.
\ No newline at end of file
+You can view and reset the application credentials of a client application after you approve it to access your API. When calling the API, the client application owner needs to pass these credentials if the API is protected with policies.
```

Comment:
Not all policies require client ID and secret. Currently only Client ID enforcement, and Rate Limit/Throttling SLA policies require them.

Reply:
Changed the content! Thanks!